code-studio/src/js/levels/multi.js

```diff
@@ -3,7 +3,7 @@ import $ from 'jquery';
 
 window.levelGroup = window.levelGroup || {levels: {}};
 
-var Multi = window.Multi = function (levelId, id, app, standalone, numAnswers, answers, answersFeedback, lastAttemptString) {
+var Multi = window.Multi = function (levelId, id, app, standalone, numAnswers, answers, answersFeedback, lastAttemptString, containedMode) {
```

Comment:
this is probably at the length where we should just pass a hash of params, but that can come another day.